src/main/java/org/springframework/data/redis/cache/DefaultRedisCacheWriter.java

```diff
@@ -401,6 +382,14 @@ private static boolean shouldExpireWithin(@Nullable Duration ttl) {
 		return ttl != null && !ttl.isZero() && !ttl.isNegative();
 	}
 
+	private Expiration toExpiration(Duration ttl) {
```

Comment:
These tiny methods are unnecessary and generate a lot of noise making it harder to read the code.

Reply:
Technically, I have even seen things likes this:

```java
return connection.stringCommands().set(ByteBuffer.wrap(key), ByteBuffer.wrap(value),
         Expiration.from(ttl.toMillis(), TimeUnit.MILLISECONDS), SetOption.upsert());
```

Reply:
`toExpiration(..)` is a easy to read method, `private`, and at the end of the class.

Reply:
Additionally, these methods can be use as _Lambda's_, such as in `.map(:Function<S, T>)` functions, which are useful in the context of Optionals, Streams, etc.